src/main.ts

```diff
@@ -26,9 +26,16 @@ async function run(): Promise<void> {
     // All the keys necessary to deploy the extension
     const keys: Keys = JSON.parse(getInput("keys", { required: true }))
     // Path to the zip file to be deployed
-    const artifact = getInput("artifact")
+    const artifact = getInput("zip") || getInput("artifact")
     const versionFile = getInput("version-file")
 
+    const verbose = !!getInput("verbose")
+
+    if (verbose) {
+      // All internal logging goes to info
+      console.log = info
```

Comment:
I don't think this is necessary.

Looking at the info function's [code](https://github.com/actions/toolkit/blob/27f76dfe1afb2b7e5e679cd8e97192d34d8320e6/packages/core/src/core.ts#L290) it's merely the following: 

```
export function info(message: string): void {
  process.stdout.write(message + os.EOL)
}
```

`info` is a wrapper that writes strings to stdout, like `console.log`, but doesn't offer the other nice properties that `console.log` has (`console.log("some string", someObj)` etc) and monkey patching might be confusing from a maintanence perspective

I suggest we just don't do this